src/applications/login/containers/SignInApp.jsx

```diff
@@ -5,145 +5,46 @@ import 'url-search-params-polyfill';
 
 import AutoSSO from 'platform/site-wide/user-nav/containers/AutoSSO';
 import LoginContainer from 'platform/user/authentication/components/LoginContainer';
-import FedWarning from 'platform/user/authentication/components/FedWarning';
-import LogoutAlert from 'platform/user/authentication/components/LogoutAlert';
 
-import ExternalServicesError from 'platform/monitoring/external-services/ExternalServicesError';
-import SubmitSignInForm from 'platform/static-data/SubmitSignInForm';
-
-import {
-  isAuthenticatedWithSSOe,
-  loginGov,
-  loginGovMHV,
-  loginGovMyVAHealth,
-  loginGovCreateAccount,
-} from 'platform/user/authentication/selectors';
-import { selectProfile, isProfileLoading } from 'platform/user/selectors';
-
-import downtimeBanners from '../utilities/downtimeBanners';
+import { isAuthenticatedWithSSOe } from 'platform/user/authentication/selectors';
+import { selectProfile } from 'platform/user/selectors';
 import { EXTERNAL_APPS } from 'platform/user/authentication/constants';
 
 class SignInPage extends React.Component {
-  state = {
```

Comment:
Seems like `globalDowntime` was unused here as I could not find anywhere that actually sets it to a value. Stripped out relevant code